docs/src/_getting-started/part1.md

```diff
@@ -0,0 +1,217 @@
+
+   ```bash
+   meltano select tap-gitlab --list --all
+   ```
+  ![Meltano Version](images/part1/gif_meltano_select_all.gif)
+
+
+1. select the entities and attributes for extraction using [`meltano select`](/reference/command-line-interface#select):
+
+   ```bash
+   meltano select tap-gitlab commits id
+   meltano select tap-gitlab commits project_id
+   meltano select tap-gitlab commits created_at
+   meltano select tap-gitlab commits author_name
+   meltano select tap-gitlab commits message
+
+   # Exclude matching attributes of all entities
+   meltano select tap-gitlab --exclude "*" "*_url"
+   ```
+
+   As you can see in the example, entity and attribute identifiers can contain wildcards (`*`) to match multiple entities or attributes at once.
+
+   This will add the [selection rules](/concepts/plugins#select-extra) to your [`meltano.yml` project file](/concepts/project#plugin-configuration):
+
+   ```yml
+   plugins:
+     extractors:
+       - name: tap-gitlab
+         variant: meltanolabs
+         pip_url: git+https://github.com/MeltanoLabs/tap-gitlab.git
+   environments:
+     - name: dev
+       config:
+         plugins:
+           extractors:
+             - name: tap-gitlab
+               config:
+                 projects: meltano/meltano meltano/tap-gitlab
+                 start_date: "2022-03-01T00:00:00Z"
+               select:
+                 - commits.id
+                 - commits.project_id
+                 - commits.created_at
+                 - commits.author_name
+                 - commits.message
+                 - tags.*
+                 - "!*.*_url"
+   ```
+
+   Note that exclusion takes precedence over inclusion, always.
+
+1. Run [`meltano select --list`](/reference/command-line-interface#select) to double-check your selection:
+
+   ```bash
+   meltano select tap-gitlab --list
+   ```
+
+## Add a dummy loader to dump the data into JSON
+To test that the extraction process works, we add a JSON target. 
+
+1. Add the JSON targt using ```meltano add loader target-jsonl```.
+![Meltano Version](images/part1/gif_meltano_add_loader_jsonl.gif)
+
+
+This target requires zero configuration, it just outputs the data into a timestamped ```jsonl``` file.
+
+## Do a test run to verify the extraction process works
+
+Now that [your Meltano project](#create-your-meltano-project), [extractor](#add-an-extractor-to-pull-data-from-a-source), and [dummy loader](#add-a-loader-to-send-data-to-a-destination) are set up, we can test run the extraction process.
+
+There's just one step here: run your newly added extractor and jsonl loader in a pipeline using [`meltano run`](/reference/command-line-interface#run):
+
+```bash
+$ meltano run tap-gitlab target-jsonl
+```
+![Meltano Version](images/part1/gif_meltano_run.gif)
+
+You should see data flowing from your source into the jsonl file.
+You can verify that it worked by looking inside the newly created file called ```output/commits.jsonl```. 
```

Comment:
Perhaps here we can keep them on the command line and have them run `cat output/commits.jsonl`?

Reply:
agree.